controller/src/main/java/org/apache/rocketmq/controller/Controller.java

```diff
@@ -106,4 +107,11 @@ CompletableFuture<RemotingCommand> alterSyncStateSet(
      * Get the remotingServer used by the controller, the upper layer will reuse this remotingServer.
      */
     RemotingServer getRemotingServer();
+
+    /**
+     * Clean controller broker data, the broker which  shut down or offline
```

Comment:
“Clean controller broker data, the broker which  shut down or offline” -> "Clean controller broker data"